### PR TITLE
ENG-445: Pass the repo option when calling the generate map command.

### PR DESCRIPTION
**Pull request by cubes**
### What

The `map` command needs to know the repo it is working on in order
to perform incremental dependency map generation. Update the
invocation of this command to include the repo origin.

### CODE REVIEW

File: src/action.js [hunk 1]
```diff
@@ -182,6 +182,9 @@ async function runCodeseeMap(config, excludeLangs) {
   if (config.apiToken) {
     args.push("-a", config.apiToken);
   }
+
+  args.push("-r", `https://github.com/${config.origin}`);
```
Comment (jleven) [inline src/action.js:186]: Why do all the other args need to be wrapped in an `if (config.origin) {` but not this one?

**Review by jleven — APPROVED** [review]
Assuming my one concern is not a concern, looks good to me!

File: src/action.js (hunk 1 above)
Comment (cubes) [reply src/action.js:186]: The other options may be falsey. `config.origin` is always set to the value of `process.env.GITHUB_REPOSITORY` which is always a truthy string.

Comment (jleven) [reply src/action.js:186]: That's true today, but may not be true forever, right? Regardless, thanks for explaining this to me. I'm satisfied.

**Review by robinjoseph08 — APPROVED** [review]

**Pull request merged**